corehq/apps/couch_sql_migration/couchsqlmigration.py

```diff
@@ -581,8 +581,8 @@ def _copy_form_properties(domain, sql_form, couch_form):
         sql_form.orig_id = getattr(couch_form, 'orig_id', None)
 
     sql_form.edited_on = getattr(couch_form, 'edited_on', None)
-    if couch_form.is_deprecated or couch_form.is_deleted:
```

Comment:
I don't recall why I added this in the first place but it doesn't seem quite correct. Perhaps it should have been it's own check. Either way it's not very important for deleted forms.